src/renderer/contexts/AlertBoxContext.tsx

```diff
@@ -67,6 +67,49 @@ const EMPTY_MESSAGE: InternalMessage = {
 
 const ALERT_FOCUS_ID = 'alert-focus-button';
 
+const pickAlertIcon = (type: AlertType) => {
```

Comment:
The return type of this function is incorrect. It's `JSX.Element | undefined` right now, but you likely intended it to be `JSX.Element`. This is because just calling `assertNever(type)` won't communicate the TS that this path never returns.

To fix this, please add a return type annotation to this function and make it `return assertNever(type)`.
```suggestion
const pickAlertIcon = (type: AlertType): JSX.Element => {
```